### ISSUE TITLE
Chests links breaking after server restarts.

### ISSUE THREAD

**Issue by ChristianRansom** [Codisimus/PhatLoots#50]
We are trying to set up phatloots on our zombie apocalypse server. Players can grief the map and collect resources from it since it resets every few hours. But when the map resets, all the links to the chests break. 

When trying to relink them, I get this error: http://pastebin.com/3ig0FFri

Let me know if theres any other information I should provide. Phatloots looks like an amazing plugin, and we’d really like to incorporate it on our server if we could get it to work.

**Comment by tara81** [comment]
is the map deleting then recreating ? If so that might explain it.  

Long ago, phatloots wouldn't delete after the map got deleted, and caused lots of errors, and wouldnt load no more, so now it auto unlinks from a map that no longer exists.

**Comment by tara81** [comment]
If its just a region reset like a schematic worldguard thing, then im not sure.  Maybe allow the chests to not be broken ?

**Comment by ChristianRansom** [comment]
Yeah that is likely the problem. One of the worlds gets deleted and replaced, however we have other worlds which do no that the chest links also break. 

I’m wondering if theres any possible way around this?

**Comment by tara81** [comment]
Sorry I really don't know other then waiting for cody to come back =( The author has been gone for 3 months now :(
